gnome-initial-setup/pages/live-chooser/eos-setup-live-user

```diff
@@ -37,34 +37,59 @@ EOS_INSTALLER = 'eos-installer.desktop'
 GDM_APPS_DIR = '/usr/share/gdm/greeter/applications'
 EOS_INSTALLER_PATH = os.path.join(GDM_APPS_DIR, EOS_INSTALLER)
 
-SCHEMA = 'org.gnome.shell'
+SHELL_SCHEMA = 'org.gnome.shell'
 ICON_GRID_LAYOUT = 'icon-grid-layout'
 TASKBAR_PINS = 'taskbar-pins'
 
+GS_SCHEMA = 'org.gnome.software'
+ALLOW_UPDATES = 'allow-updates'
+
+class SettingsUpdates(object):
```

Comment:
I agree, less (if it makes sense) is better. I just did want to be too aggressive